usr.sbin/kbdcontrol/kbdcontrol.c

```diff
@@ -1085,14 +1086,15 @@ set_keyrates(char *opt)
 				break;
 		d = n;
 		for (n = 0; n < nrepeats - 1; n++)
-			if (repeat <= repeats[n])
+			if (repeat <= rates[n])
 				break;
 		r = n;
 	}
 
 	arg[0] = delay;
 	arg[1] = repeat;
 	if (ioctl(0, KDSETREPEAT, arg)) {
+		warn("fallback, setting keyboard rate via legacy interface (KDSETRAD), will be removed soon");
```

Comment:
How does one avoid this message? The message itself should have better instructions. The legacy interface won't be completely removed, just compiled out by default soon. I don't see the path to not get this message as well: if we're going to say it all the time, it's kinda pointless.

Reply:
As far as I can see, even the darkest legacy ATKBC, ADB (lol) and all flavors of custom GPIO keyboards support the new interface. 

Therefore, the message is only for people with real niche custom driver (if any).  With the static mapping tables in the header and seven live code examples, it is not that complicated to migrate.

How often do you call KBDControl and adjust the repeat rate parameter manually during the day back and forth ( on a legacy non-supported custom keyboard driver device ) that necessitates the need to suppress this message? 

Being annoying is the whole point of that line. If this is a problem, I can move it into a (silent) comment.